uint8 YoloV8_Tracker = 0
uint8 Pose_Estimation = 1
uint8 Image_Captioning = 2
uint8 Face_Recognition = 3

int32 type